privacy_evaluator/attacks/sample_attack.py

```diff
@@ -26,12 +25,13 @@ def __init__(
         """
         super().__init__(target_model, x_train, y_train, x_test, y_test)
 
-    def attack(self, *args, **kwargs) -> Tuple[np.ndarray, ...]:
+    def attack(self, x: np.ndarray, y: np.ndarray, *args, **kwargs) -> np.ndarray:
         """Performs the attack on the target model.
 
+        :param x: Input data to attack.
+        :param y: True labels for x.
```

Comment:
Probably at all places where you reference y, you should specify (comment and raise assertion errors), what shape of labels do you expect. Do you want one-hot encoded labels in the shape of (number_samples, number_classes), or flat labels (number_samples, _)

Reply:
I am on it. It's not yet pushed but just for the others so we do not double fix this.

Reply:
I have added some general validators and added them to all attacks.